stylus/components/panels.styl

```diff
@@ -0,0 +1,38 @@
+@require '../../stylus/utilities/spaces.styl'
+@require 'settings/palette.styl'
+@require 'settings/breakpoints.styl'
+
+$panel-group
+    display         flex
+    flex-direction  row
+    justify-content flex-start
+    align-items     stretch
+    box-sizing      border-box
+
+$panel-main
+    flex-basis 65%
+    flex-grow 1
+    flex-shrink 0
+    overflow auto
+
+$panel-side
+    flex-basis 35%
```

Comment:
If I remember correctly some browsers don't have the same default values for `shrink` and/or `grow` and you might as well write them explicitely with `flex 0 0 35%`